analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -723,29 +728,61 @@ def add_arguments_to_parser(parser):
         func=main, func_process_config_file=cmd_config.process_config_file)
 
 
-def __get_skip_handler(args) -> skiplist_handler.SkipListHandler:
+def get_source_file_paths(
+    file_filters: List[str],
+    compile_commands: tu_collector.CompilationDB,
+    header_file_extensions=(
+        '.h', '.hh', '.H', '.hp', '.hxx', '.hpp', '.HPP', '.h++', '.tcc')
+) -> List[str]:
     """
-    Initialize and return a skiplist handler if
-    there is a skip list file in the arguments or files options is provided.
+    Returns a list of source files for existing header file otherwise returns
+    with the same file path expression.
     """
-    skip_file_content = ""
+    file_paths = []
```

Comment:
```suggestion
    file_paths = set()
```

Reply:
I used list to keep the order of the items. I added a comment to highlight it.